apps/gitlab/docker-compose.yml

```
@@ -0,0 +1,49 @@
+version: '3.6'
+services:
+  gitlab:
+    image: gitlab/gitlab-ce:17.4.1-ce.0
+    container_name: gitlab
+    hostname: ${APP_DOMAIN}
+    restart: always
+    environment:
+      https: false
+      GITLAB_OMNIBUS_CONFIG: |
+        external_url 
```

yamllint (1.29.0-1)

[error] trailing spaces

(trailing-spaces)